### PR TITLE
Add disclaimer about playing media files

### PR DESCRIPTION
**Pull request by Sophist-UK**
**Summary**

This is a…
 - [ ] Correction
 - [ ] Addition
 - [ ] Restructuring
 - [x] Minor / simple change (like a typo)
 - [ ] Other

**Reason for the Change**

Other tools can play media files and retag from Musicbrainz.

**Description of the Change**

Add disclaimer about playing media files

### CODE REVIEW

File: source/introduction.rst [hunk 1]
```diff
@@ -46,9 +46,11 @@ in a naming script.
 ...calculate and submit a music file's AcoustID fingerprint to the `AcoustID database <https://acoustid.org/>`_.
 
 
-Picard Cannot...
+Picard cannot...
 ----------------
 
+...play music files or videos.
```
Comment (rdswift) [inline source/introduction.rst:52]: You're right that Picard can't play videos, but there is a music file player built in.  Or am I missing something?

image

**Comment by rdswift** [comment]
Also, can you please set the merge target to the `development` rather than the `master` branch?  Either that or perhaps I should create a `publish` branch and publish from there, and use the `master` branch for staging since it ends up as the default target for pull requests.  Less confusing for contributors that way.

**Comment by Sophist-UK** [comment]
I wasn't confused before but now I am. What is wrong with using master as ... um ... the master branch to merge PRs to **and** to publish from?

**Comment by rdswift** [comment]
> I wasn't confused before but now I am. What is wrong with using master as ... um ... the master branch to merge PRs to **and** to publish from?

Nothing wrong with it.  That's likely the simplest.  I was just using the `development` branch for staging changes because the system is currently set up to publish each time there is a change to the master branch, and it was getting quite a workout during initial development.  That shouldn't be a problem now.  I'll make the necessary changes.

File: source/introduction.rst (hunk 1 above)
Comment (phw) [reply source/introduction.rst:52]: Maybe change it to "play all audio formats"?

**Pull request closed without merging**